src/store/modules/accounts/index.js

```diff
@@ -57,8 +57,11 @@ export default {
       return dispatch(`${getModule(active).name}/sign`, data);
     },
 
-    signTransaction({ getters: { active, getModule }, dispatch }, txBase64) {
-      return dispatch(`${getModule(active).name}/signTransaction`, txBase64);
+    signTransaction({ getters: { active, getModule }, dispatch }, { txBase64, opt }) {
+      if (opt && opt.fromAccount) {
+        return dispatch(`${getModule(active).name}/signTransactionFromAccount`, { txBase64, opt });
+      }
+      return dispatch(`${getModule(active).name}/signTransaction`, { txBase64, opt });
     },
```

Comment:
why you send "opt"? signTransactionFromAccount need only "txBase64"

Then signTransactionFromAccount is not nesesary becouse you can do that:
```suggestion
     const payload = { txBase64 }
     if (!(opt && opt.fromAccount)){
     payload.opt = opt
     }
      return dispatch(`${getModule(active).name}/signTransaction`, payload);
    },
```